.NET/Microsoft.Recognizer.Text.TimeZone/Microsoft.Recognizers.Text.TimeZone.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All recognizers must follow the same project model. Please take a look at the others and convert accordingly.
This is also necessary as the recognizers need to be built for different framework targets.